### PR TITLE
Enforce mvn 3.9.8+ or mvnd 1.0.0+

### PR DESCRIPTION
**Pull request by wendigo**
This makes it explicit what is the actual requirement

### CODE REVIEW

**Review by mosabua — APPROVED** [review]
Looks good but lets fix grammar in the commit message.

File: pom.xml [hunk 1]
```diff
@@ -152,6 +152,8 @@
         <air.check.skip-jacoco>true</air.check.skip-jacoco>
         <air.java.version>22.0.0</air.java.version>
         <air.javadoc.lint>syntax,reference,html,-missing</air.javadoc.lint>
+        <!-- require at least maven 3.9.8 or mvnd 1.0.0+ -->
+        <air.maven.version>3.9.8</air.maven.version>
```
Comment (findepi) [inline pom.xml:156]: airbase defaults air.maven.version to 3.6.3
what's the benefit of setting this differently in trino?

Comment (wendigo) [reply pom.xml:156]: I can set this in airbase

Comment (wendigo) [reply pom.xml:156]: https://github.com/airlift/airbase/pull/414

**Pull request closed without merging**